src/common/models/repo.go

```diff
@@ -38,6 +45,48 @@ type RepoRecord struct {
 	UpdateTime   time.Time `orm:"column(update_time);auto_now" json:"update_time"`
 }
 
+// FilterByBlobDigest filters the repositories by the blob digest
+func (r *RepoRecord) FilterByBlobDigest(ctx context.Context, qs orm.QuerySeter, key string, value interface{}) orm.QuerySeter {
+	digest, ok := value.(string)
```

Comment:
If you only accept a `string` as arg, why not directly declare the value to a `string` type? The caller needs to guarantee that the passing parameter should be a string value.

Reply:
The method `FilterByXXX` is an interface defined for the ORM query, it isn't only for the repository